src/components/add-journey/FlightTimeToggle.tsx

```diff
@@ -0,0 +1,29 @@
+import { classNames } from '@/utils/styling';
+import { useTranslations } from 'next-intl';
+
+import Switch from 'react-switch';
+
+type Props = {
+  isArrival: boolean;
+  setIsArrival: (isArrival: boolean) => void;
+  className?: string;
+};
+
+export const FlightTimeToggle: React.FC<Props> = ({ className, isArrival, setIsArrival }) => {
```

Comment:
Let's rename this to `DepartureOrArrivalTimeToggle` since we're not dealing with Flights/Planes ;)

Reply:
oh this one was hilarious!